keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/AdminControllerTests.kt

```diff
@@ -1,95 +0,0 @@
-package com.netflix.spinnaker.keel.rest
```

Comment:
Removed because we're no longer using the same methods on the admin controller, so this isn't relevant. There are tests for the `Cleaner` methods, so no need to update the tests here.